File: tehnohem-api/DTO/IncomingInvoiceDTO.cs
using tehnohem_api.Model.Enums;
using tehnohem_api.Model;

namespace tehnohem_api.DTO
{
    public class IncomingInvoiceDTO
    {
        public string invoiceID { get; set; }
        public string SupplierID { get; set; }
        public DateOnly Date { get; set; }
        public float TotalValue { get; set; }
        public float TotalValueOfPDV { get; set; }
        public float TotalValueWithoutPDV { get; set; }

        public ICollection<IncomingInvoiceItemDTO> InvoiceItems { get; set; }


    }
}

File: tehnohem-ui/src/app/shared/model/invoices/incomingInvoiceDTO.ts
import { Company } from "../company";
import { Raw } from "../raw";
import { RawToSell } from "../rawToSell";

export interface IncomingInvoiceDTO{
    invoiceID: string;
    date : string;
    supplierID: string;
    invoiceItems : RawToSell[];
    totalValueOfPdv: number;
    totalValueWithoutPdv: number;
    totalValue : number;
}